{
  "comments": [
    {
      "key": {
        "uuid": "bab921fb_85eff3af",
        "filename": "ci-scripts/dlrnapi_promoter/test_get_latest_hashes.py",
        "patchSetId": 3
      },
      "lineNbr": 73,
      "author": {
        "id": 84
      },
      "writtenOn": "2018-06-12T16:38:34Z",
      "side": 1,
      "message": "This test is not working.",
      "range": {
        "startLine": 46,
        "startChar": 0,
        "endLine": 73,
        "endChar": 47
      },
      "revId": "106e9a0cd07e602ce0577ef0a4653774376bc2a1",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "bab921fb_808da17d",
        "filename": "ci-scripts/dlrnapi_promoter/test_get_latest_hashes.py",
        "patchSetId": 3
      },
      "lineNbr": 73,
      "author": {
        "id": 517
      },
      "writtenOn": "2018-06-13T08:20:35Z",
      "side": 1,
      "message": "Fixed promoter, there were a defect in case there are no old hashes.",
      "parentUuid": "bab921fb_85eff3af",
      "range": {
        "startLine": 46,
        "startChar": 0,
        "endLine": 73,
        "endChar": 47
      },
      "revId": "106e9a0cd07e602ce0577ef0a4653774376bc2a1",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "bab921fb_a5f2f798",
        "filename": "tox.ini",
        "patchSetId": 3
      },
      "lineNbr": 23,
      "author": {
        "id": 84
      },
      "writtenOn": "2018-06-12T16:38:34Z",
      "side": 1,
      "message": "The dlrnapi_promoter script looks like it is not py3 compatible. I get an error in py3 when importing get_latest_hashes related to the \"from sets import Set\"",
      "range": {
        "startLine": 20,
        "startChar": 0,
        "endLine": 23,
        "endChar": 35
      },
      "revId": "106e9a0cd07e602ce0577ef0a4653774376bc2a1",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "bab921fb_80e4813b",
        "filename": "tox.ini",
        "patchSetId": 3
      },
      "lineNbr": 23,
      "author": {
        "id": 517
      },
      "writtenOn": "2018-06-13T08:20:35Z",
      "side": 1,
      "message": "Fixed, we don\u0027t need Set, normal set is enough and works both py27 and py35, also ConfigParser have to be configparser.",
      "parentUuid": "bab921fb_a5f2f798",
      "range": {
        "startLine": 20,
        "startChar": 0,
        "endLine": 23,
        "endChar": 35
      },
      "revId": "106e9a0cd07e602ce0577ef0a4653774376bc2a1",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    }
  ]
}